metrics/handler.go

```diff
@@ -32,6 +33,9 @@ type metricsHandler struct {
 func (mh *metricsHandler) Start(mux *http.ServeMux, addr string) error {
 	mh.logger.Info("setup metrics collection")
 
+	runtime.SetBlockProfileRate(1000)
```

Comment:
@nivBlox not sure it's relevant.. if it does it should be In a different PR

Reply:
@amir-blox is moving it to differ pr

Reply:
Added in https://github.com/bloxapp/ssv/pull/215